lib/salus/scanners/lang_version.rb

```diff
@@ -0,0 +1,75 @@
+require 'salus/scanners/base'
+require 'json'
+
+module Salus::Scanners
+  class LangVersion < Base
+    class SemVersion < Gem::Version; end
+
+    def run
+      version = SemVersion.new(lang_version)
+      min_version = SemVersion.new(@config['min_version']) if @config['min_version']
+      max_version = SemVersion.new(@config['max_version']) if @config['max_version']
+
+      if min_version && (version < min_version)
```

Comment:
May wanna check that the version checks work with things like
`2.8.1 < 2.10.1`

Reply:
This should be covered by these unit tests: https://github.com/coinbase/salus/pull/493/files#diff-c44cfac3170a68bb6dd0c3fe059f5199da9d0ca21f1170dc579686472f12c30eR41-R77